test/components/labelTests.ts

```diff
@@ -189,4 +189,11 @@ describe("Labels", () => {
     var testLabel = new Plottable.Components.Label("testing label");
     assert.throws(() => testLabel.padding(-10), Error, "Cannot be less than 0");
   });
+
+  it("Label can takes number as displayText", () => {
+    assert.doesNotThrow(() => {
+      var label = new Plottable.Components.Label(1);
+      assert.strictEqual(label.text(), "1", "text equal to input number");
+    }, Error, "Label does not crash on invalid values");
```

Comment:
This is not an invalid value anymore, so the message should say "Label does not crash on numeric values"